### PR TITLE
feat: add orchestration efficiency evidence scenario

### PR DESCRIPTION
**Pull request by Phelan164**
## What Changed

Adds an `orchestration-efficiency-evidence` scenario to the evaluator-RAG prototype corpus.

The scenario:

- requires a dependency graph and exclusive write ownership before delegation;
- compares matched single-agent and orchestrated controls with the same acceptance evaluator;
- counts aggregate controller, worker, and retry cost instead of controller-only telemetry;
- preserves failures, conflicts, integration rework, retries, and human corrections;
- rejects general efficiency claims from one overlapping smoke run;
- requires repeated sequential pairs with alternated order before a task-class recommendation.

It includes the scenario, trace, report, verifier result, candidate playbook, documentation entry, and a regression test.

## Why This Change

ECC already has strong orchestration and context-budget guidance. The evaluator corpus did not yet encode the decision boundary for when parallel execution actually earns its coordination cost.

This contribution deliberately does not add another orchestration skill. It turns the existing guidance into a read-only evidence gate that rejects intuition-only claims such as “more agents are automatically faster or cheaper.”

The external smoke evidence is attributed and pinned to an immutable `codex-howto` commit. That experiment did **not** establish an efficiency result; its missing aggregate token telemetry and overlapping variants are the reason this scenario requires stronger controls.

## Testing Done

- [x] Manual fixture and diff review completed
- [x] `npm run lint`
- [x] `node tests/docs/evaluator-rag-prototype.test.js` — 14 passed
- [x] `node tests/lib/orchestration-session.test.js` — 8 passed
- [x] All new JSON fixtures validated with `jq -e`
- [x] `git diff --check`
- [x] Edge cases considered: unmatched controls, controller-only cost, failed attempts omitted, overlapping runs, fixed order, and unavailable telemetry
- [ ] Full `npm test` gauntlet passes locally

The full gauntlet was attempted in Codex's restricted workspace. Existing environment-dependent tests failed when they tried to write under `~/.claude` or bind loopback servers (`EPERM`), including hook, MCP health, and plan-canvas integration cases. The focused tests and repository lint above pass.

## Type of Change

- [x] `feat:` New feature
- [ ] `fix:` Bug fix
- [ ] `refactor:` Code refactoring
- [x] `docs:` Documentation
- [x] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist

- [x] No secrets or API keys committed
- [x] JSON files validate cleanly
- [x] No shell scripts changed
- [x] Pre-commit-equivalent focused checks pass locally
- [x] No sensitive data exposed in logs or output
- [x] Follows conventional commits format

## Dependencies / Export Surface

- No dependencies, `package.json`, skills, commands, agents, hooks, CLI tools, or export surfaces changed.

## Documentation

- [x] Updated the evaluator-RAG architecture corpus list
- [x] Added a candidate playbook for the scenario
- [ ] README updated — not needed for this corpus-only addition

## Attribution

The contributed smoke record that motivated the stricter evidence gate is pinned in `scenario.json`:

https://github.com/Phelan164/codex-howto/blob/527296fa7d9ba1828d639ec000e0543c42ee2739/examples/measurements/incident-orchestration-smoke-2026-08-11.md

### CODE REVIEW

File: examples/evaluator-rag-prototype/orchestration-efficiency-evidence/scenario.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "launching workers, editing candidate repositories, publishing benchmarks, or posting announcements from this read-only evaluator run"
+  ],
+  "acceptance_gates": [
+    "dependency graph and exclusive ownership boundaries are recorded",
+    "single-agent and orchestrated controls are matched",
+    "all attempted candidates use the same acceptance evaluator",
+    "aggregate controller and worker cost is recorded or explicitly unavailable",
+    "elapsed time, retries, conflicts, integration rework, and human corrections are retained",
+    "failed candidates remain in the comparison",
+    "at least one one-run parallelism claim is rejected"
```
[the remark this answers is not in the document]
Comment (Phelan164) [reply examples/evaluator-rag-prototype/orchestration-efficiency-evidence/scenario.json:58]: Fixed in c2abbc8. The scenario now requires at least three fresh sequential pairs with alternated order before promotion, and the evaluator-RAG regression test asserts the exact gate. Focused evaluator tests (14/14), lint, JSON validation, and diff checks pass.

File: examples/evaluator-rag-prototype/orchestration-efficiency-evidence/trace.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "scripts/lib/orchestration-session.js"
+      ]
+    },
+    {
+      "phase": "retrieval",
+      "summary": "Retrieved dependency and ownership plans, matched run controls, acceptance results, aggregate token or cost telemetry, elapsed time, failures, conflicts, integration rework, retries, and human corrections.",
+      "evidence": [
+        "skills/context-budget/SKILL.md",
+        "node tests/lib/orchestration-session.test.js",
+        "contributed orchestration smoke evidence pinned to commit 527296f"
```
[the remark this answers is not in the document]
Comment (Phelan164) [reply examples/evaluator-rag-prototype/orchestration-efficiency-evidence/trace.json]: Fixed in 6daf759. The retrieval trace now stores the same full commit-pinned GitHub URL as the scenario source, and the evaluator-RAG test asserts that the external experiment URL remains present in the serialized trace. Focused corpus validation passes 14/14.

File: tests/docs/evaluator-rag-prototype.test.js [hunk 3]
```diff
@@ -409,6 +409,55 @@ test('deep analyzer evidence scenario rejects no-corpus analyzer changes', () =>
...
+  const accepted = verifier.candidates.find(candidate => candidate.candidate_id === 'controlled-paired-orchestration-evaluation');
+  const rejected = verifier.candidates.find(candidate => candidate.candidate_id === 'parallelism-implies-efficiency');
+
+  assert.ok(accepted, 'Missing accepted controlled orchestration candidate');
+  assert.ok(rejected, 'Missing rejected intuition-only orchestration candidate');
+  assert.strictEqual(accepted.decision, 'accepted');
+  assert.strictEqual(rejected.decision, 'rejected');
+  assert.strictEqual(verifier.promoted_candidate_id, accepted.candidate_id);
+  assert.ok(rejected.reasons.join('\n').includes('no matched accepted single-agent baseline'));
+  assert.ok(JSON.stringify(trace).includes(scenario.sources.find(source => source.kind === 'external_experiment').url));
```
[the remark this answers is not in the document]
Comment (Phelan164) [reply tests/docs/evaluator-rag-prototype.test.js]: Fixed in 926c596. The test now validates the external experiment source, locates the retrieval event explicitly, and requires retrieval.evidence to contain that source's exact URL. The focused evaluator-RAG corpus still passes 14/14.

**Comment by Phelan164** [comment]
Maintainer-ready status after the review fixes:

- mergeable with seven scoped files: one architecture registration, five read-only evidence artifacts, and one test file;
- focused evaluator-RAG suite passed 14/14 on the latest contributor run;
- CodeRabbit, GitGuardian, and Greptile are green;
- the last CodeRabbit assertion finding was fixed in `926c596`; no unresolved automated finding remains.

The scenario rejects orchestration-efficiency claims unless matched controls, quality gates, aggregate controller-plus-worker cost, coordination overhead, failures/rework, and repeated sequential pairs are present. It performs no external writes.

@affaan-m, this should be ready for maintainer review. I can rebase or narrow it if the evaluator corpus has moved; otherwise no further contributor action is pending.

**Comment by chenhz01** [comment]
Solid gate — the aggregate-cost and kept-failures rules kill the usual controller-only telemetry claim. One statistical gap in the accepted path: three sequential pairs establish little when run-to-run variance is high, which agent workloads are. Require the report to expose per-pair deltas and a consistency rule before a task-class recommendation:

```json
"pairDeltas": [{"pair": 1, "timeDeltaMs": null, "costDeltaUsd": null}],
"recommendationAllowedOnlyIf": "deltas sign-consistent across pairs"
```

With n=3, mixed-sign deltas should block a recommendation rather than average out — the corpus already rejects one-run claims, and mixed-direction evidence is the same failure mode at n=3. Happy to extend the scenario schema and the verifier check with this if useful.
